debug/zoomrange.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
This file doesn't seem like a good addition to the `debug/` folder. Mind removing it?